homeassistant/components/opentherm_gw/__init__.py

```diff
@@ -2,9 +2,18 @@
 import logging
 from datetime import datetime, date
 
+import pyotgw
+import pyotgw.vars as gw_vars
 import voluptuous as vol
 
 from homeassistant.components.binary_sensor import DOMAIN as COMP_BINARY_SENSOR
+from homeassistant.components.climate import DOMAIN as COMP_CLIMATE
+from homeassistant.components.opentherm_gw.const import (
```

Comment:
Please make all intra package imports relative. This will allow custom integrations to override builtin integrations.

```py
from .const import (...)
```

Reply:
I used relative imports while it was a WIP, decided to change to the absolute form to make the imports more uniform. I will change it back with the next commit.